Assets/Romey/DropdownController.cs

```diff
@@ -0,0 +1,88 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.XR.Interaction.Toolkit;
+using GVR.Phoenix.Interface;
+
+
+public class DropdownController : MonoBehaviour
+{
+    bool fontloaded = false;
+    public GameObject textAsset;
+    public GameObject locomotionsystem;
+    public ActionBasedSnapTurnProvider snapTurn;
+    public ActionBasedContinuousTurnProvider continuousTurn;
+    // Start is called before the first frame update
+    void Start()
+    {
+
+        var dropdown = transform.GetComponent<TMP_Dropdown>();
+        dropdown.options.Clear();
+
+        List<string> items = new List<string>();
+        items.Add("No turning");
+        items.Add("Snap turn");
+        items.Add("Continuous turn");
+
+        foreach (var item in items)
+        {
+            dropdown.options.Add(new TMP_Dropdown.OptionData() { text = item });
+        }
+
+        dropdown.onValueChanged.AddListener(delegate { DropdownItemSelected(dropdown); });
+        snapTurn = locomotionsystem.GetComponent<ActionBasedSnapTurnProvider>();
+        continuousTurn = locomotionsystem.GetComponent<ActionBasedContinuousTurnProvider>();
+
+        
+
+    }
+
+    void DropdownItemSelected(TMP_Dropdown dropdown)
+    {
+
+        if (dropdown.value == 0) 
```

Comment:
I feel like a switch statement would be better here instead of multiple ifs

Reply:
Discarded for now. I will improve it on a different branch.